pandas/tests/strings/test_strings.py

```diff
@@ -17,6 +19,27 @@
 import pandas._testing as tm
 
 
+@pytest.fixture(
+    params=[
+        "object",
+        "string",
+        pytest.param(
+            "arrow_string", marks=td.skip_if_no("pyarrow", min_version="1.0.0")
+        ),
+    ]
+)
+def any_string_dtype(request):
+    """
+    Parametrized fixture for string dtypes.
+    * 'object'
+    * 'string'
+    * 'arrow_string'
+    """
+    from pandas.core.arrays.string_arrow import ArrowStringDtype  # noqa: F401
+
+    return request.param
```

Comment:
A similar fixture does not yet exist?

Reply:
not yet. this is being added to a couple of PRs and can be promoted to a higher level conftest once one of the PRs has been merged.

Reply:
OK, sounds good